src/components/Sidebar/chat/chatInput.tsx

```diff
@@ -40,14 +57,17 @@ export function SidebarInput({
   return (
     <div className="cdx-fixed cdx-bottom-0 cdx-left-0 cdx-right-0 cdx-flex cdx-flex-col ">
       <div className="cdx-flex cdx-items-center cdx-justify-between">
-        {!chatIsEmpty && (
+        {!chatIsEmpty ? (
           <button
             onClick={clearMessages}
             className="cdx-rounded-full cdx-h-10 cdx-w-10 cdx-grid cdx-place-items-center cdx-text-center cdx-bg-blue-500 hover:cdx-bg-blue-700 cdx-text-white cdx-m-2"
           >
             <GiMagicBroom size={18} className="mx-auto" />
           </button>
-        )}
+        ) : <div></div>}
```

Comment:
instead of this, we can do 

```typescript
 {!chatIsEmpty && (
          <button
            onClick={clearMessages}
            className="cdx-rounded-full cdx-h-10 cdx-w-10 cdx-grid cdx-place-items-center cdx-text-center cdx-bg-blue-500 hover:cdx-bg-blue-700 cdx-text-white cdx-m-2"
          >
            <GiMagicBroom size={18} className="mx-auto" />
          </button>
        )}
```

Reply:
we are using flex between to algin clear btn and history box and that's why I use empty div here